std/machines/shift_bb.asm

```diff
@@ -0,0 +1,70 @@
+use std::utils::unchanged_until;
+use std::utils::cross_product;
+use std::convert::int;
+
+// Shift for single bytes using an exhaustive table
+// TODO this way, we cannot prove anything that shifts by more than 31 bits.
+machine ByteShiftBB with
+    latch: latch,
+    operation_id: operation_id,
+    degree: 65536
+{
+    // P_C1 and P_C2 are both 16 bit limbs of P_C, where P_C1 is the less significant limb.
```

Comment:
Don't we usually have big endian ordering?

Reply:
We had little endian in binary bb machine, but as just discussed with Leo both should be good.